### ISSUE TITLE
how to finetune TIMIT model with my own speakers audios?

### ISSUE THREAD

**Issue by MuruganR96** [HarryVolek/PyTorch_Speaker_Verification#9]
@HarryVolek Sir, i have model for our **PyTorch_Speaker_Verification** ( epochs: 950 Max training speaker epoch for TIMIT)
sir i have a doubt. In that pytorch model, we will finetune(transfer learning/ Retrain) with my own speakers audios again.
Is it possible or not sir? which way to implement this?

AND another doubt, if we have only 10-20 speakers audio utterence, will it gives good accuracy or not?
(or)
in this 10-20 speakers audio utterence to build own model, will it gives good accuracy or not?

In that Voice Activity Detection how can we achieve "Wake word detection" sir? then how can i do this in dynamic way?

i am more concentrate on this. thank you for your initiative.
thank you lots @HarryVolek sir
